### DISCUSSION TITLE
Inaccurate last activity in notifications list

### DISCUSSION THREAD

**Discussion by EricFromCanada** [community/community#120503] [Mobile]
Noticed that the issue in the attached image says that I requested a review from myself, but that's not right. If I tap into the issue it states correctly that I requested a review from someone else. 

![image](https://github.com/community/community/assets/3324775/18cc1cd8-79c8-47cc-b4ef-c453d7fccb01)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.158.0 (128225528)
Device: iPhone 13 Pro (iOS 17.3.1)
</details>

**Comment by HenSquared** [comment]
Thanks @EricFromCanada, I've created an issue in our iOS repository to track this issue.
